app/jobs/compact_uploads_job.rb

```diff
@@ -5,6 +5,22 @@
 class CompactUploadsJob < ApplicationJob
   queue_as :default
 
+  def self.enqueue_some(ratio: 0.25, maximum: 5) # rubocop:disable Metrics/AbcSize
+    streams = Stream.include(:statistic).default.select do |stream|
+      next unless stream.statistic.unique_record_count && stream.statistic.record_count
```

Comment:
Should this be `stream.statistic&.unique_record_count` in case `stream.statistics` is `nil`?